Core/src/org/sleuthkit/autopsy/logicalimager/dsp/AddLogicalImageTask.java

```diff
@@ -268,22 +281,25 @@ void cancelTask() {
     }
 
     private Map<String, Long> imagePathsToDataSourceObjId(Map<Long, List<String>> imagePaths) {
-        Map<String, Long> imagePathToObjIdMap = new HashMap<>();
+        Map<String, Long> imagePathToObjId = new HashMap<>();
         for (Map.Entry<Long, List<String>> entry : imagePaths.entrySet()) {
             Long key = entry.getKey();
             List<String> names = entry.getValue();
             for (String name : names) {
-                imagePathToObjIdMap.put(name, key);
+                imagePathToObjId.put(name, key);
             }
         }
-        return imagePathToObjIdMap;
+        return imagePathToObjId;
     }
 
     @Messages({
         "# {0} - line number", "# {1} - fields length", "# {2} - expected length", "AddLogicalImageTask.notEnoughFields=File does not contain enough fields at line {0}, got {1}, expecting {2}",
-        "# {0} - target image path", "AddLogicalImageTask.cannotFindDataSourceObjId=Cannot find obj_id in tsk_image_names for {0}"
+        "# {0} - target image path", "AddLogicalImageTask.cannotFindDataSourceObjId=Cannot find obj_id in tsk_image_names for {0}",
+        "# {0} - file number", "# {1} - total files", "AddLogicalImageTask.addingInterestingFile=Adding interesting file {0} of {1}"
```

Comment:
I'd change this to "Adding interesting files (x of y)" or a slash in place of the "of" so it doesn't read like you're specifically adding artifact 200. Same with the adding extracted files message.